### DISCUSSION TITLE
Does anybody know how to help me with that!

### DISCUSSION THREAD

**Discussion by UnordinarySheep** [KosmosisDire/obsidian-webpage-export#518] [General]
https://github.com/oleeskild/obsidian-digital-garden/discussions/640

**Comment by KosmosisDire** [comment]
This is a completely different plugin, so this isn't really the right place to ask. Are you wanting to use the webpage html export plugin?

**Comment by UnordinarySheep** [comment]
no but since i saw this one manged to load canvas i was wondering how did you make it do that and how can i make it work with 11ty composition. Potentially fix my other problem with the excalidrawing too

**Comment by KosmosisDire** [comment]
My plugin loads canvas files, but you can't make that work with the digital garden plugin unless they add support for it themselves. Same with excalidraw, my plugin supports excalidraw but I can't do anything to fix excalidraw on the digital garden plugin unfortunately.

If you need help getting this plugin to work I can continue to help (I know you were having trouble getting it to work before, but you never replied to my last comment), but I can't do much to help with a different plugin.

**Reply by UnordinarySheep** [reply]
i managed to fix it. I use your plugin to load my canvas. Its really good. Im just working with all of them and trying to build like a middle version combined version of them so it will be really easy for me. just get  all the good features from them.

**Reply by KosmosisDire** [reply]
Oh good, glad you got it working! :)
